build-support/bin/check_inits.py

```diff
@@ -7,6 +7,8 @@
 
 from common import die
 
+from pants.util.strutil import softwrap
```

Comment:
This means that you now can only run the script via `./pants run`. I don't think that's worth it